app/dto/person.js

```diff
@@ -12,6 +12,35 @@ const personDto = (person) => ({
   }
 })
 
+const personAndDogsDto = (personAndDogs) => ({
+  firstName: personAndDogs[0].person.first_name,
+  lastName: personAndDogs[0].person.last_name,
+  birthDate: personAndDogs[0].person.birth_date,
+  personReference: personAndDogs[0].person.person_reference,
+  address: {
+    addressLine1: personAndDogs[0].person.addresses[0].address.address_line_1,
+    addressLine2: personAndDogs[0].person.addresses[0].address.address_line_2,
+    town: personAndDogs[0].person.addresses[0].address.town,
+    postcode: personAndDogs[0].person.addresses[0].address.postcode,
+    country: personAndDogs[0].person.addresses[0].address.country.country
+  },
+  contacts: personAndDogs[0].person.person_contacts,
+  dogs: personAndDogs.map(x => ({
```

Comment:
As we are only reading the dog object could we not just call map on `personAndDogs.dogs` and then rename `x` in the map to `dog`?

Reply:
Yep, good point. Will change